src/tests/e2e/mocks/drive.js

```diff
@@ -78,13 +78,16 @@ var data = {
   }
 }
 
-var storage = window.localStorage
+var storage
 
-if (!storage.getItem('data')) {
-  // prevent refresh from clearing data
-  storage.setItem('data', JSON.stringify(data))
-}
+function initStorage () {
+  storage = window.localStorage
```

Comment:
#1502 States 'Do not use window in e2e tests in mock drive.js', please update issue description.

Reply:
updated